src/Voron/Data/Tables/Table.cs

```diff
@@ -704,59 +921,56 @@ internal FixedSizeTree GetFixedSizeTree(Tree parent, Slice name, ushort valSize,
             return tree;
         }
 
-        private long AllocateFromSmallActiveSection(TableValueBuilder builder, int size)
+        private void CreateNewActiveSection()
         {
-            if (ActiveDataSmallSection.TryAllocate(size, out long id) == false)
+            ushort maxSectionSizeInPages =
```

Comment:
Something that came to my mind. We need to check if moving between 32 and 64 bits work.

Reply:
It *shouldn't* matter, but we are taking more memory in this case, I think